.github/workflows/test.yml

```diff
@@ -0,0 +1,56 @@
+name: Test
```

Comment:
We might need to add more detail here

Reply:
@suho  @luongvo Suggested me to name this workflow to `Test`. 😄 Thus, I rename it to according to the filename. Do you have any suggestions? 🙏🏻